Danki2/Assets/Scripts/Abilities/AbilityObjects/DaggerObject.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using UnityEngine;
+
+public class DaggerObject : ProjectileObject
+{
+    public static void Fire(Actor caster, Action<GameObject> collisionCallback, float speed, Vector3 position, Quaternion rotation)
```

Comment:
Do we want to have the Fire method enforced through an interface? Can't currently see any enforcement for this, is there a reason for that?

Reply:
The short answer is, you can't have static methods on an interface (or abstract class). I spent quite a long time trying to find a better way of doing this.

Reply:
Fair, as long as it was properly seen to.